include/TPP/Dialect/Tpp/TppOps.td

```diff
@@ -214,4 +261,56 @@ def Tpp_BrgemmOp : Tpp_Op<"brgemm"> {
   let hasVerifier = 1;
 }
 
+
+//===----------------------------------------------------------------------===//
+// VNNI_BrgemmOp
+//===----------------------------------------------------------------------===//
+
+def Tpp_VNNI_BrgemmOp : Tpp_Op<"vnni_brgemm"> {
+  let summary = "Performs batch reduced matrix multiplication of two inputs with the first input in VNNI format";
```

Comment:
format better pls

Reply:
This doesn't take a newline argument so I am forced to type the whole statement in one line.